extension/iceberg/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+if (WIN32)
+    set(DuckDB_USE_STATIC_LIBS ON)
+else ()
+    set(DuckDB_USE_STATIC_LIBS OFF)
+endif ()
+find_package(DuckDB REQUIRED)
+
+include_directories(
+        ${PROJECT_SOURCE_DIR}/src/include
+        src/include
+        ${PROJECT_SOURCE_DIR}/extension/duckdb/src/include
+        ${PROJECT_SOURCE_DIR}/extension/httpfs/src/include
+        ${PROJECT_SOURCE_DIR}/extension/delta/src/include
+        ${DuckDB_INCLUDE_DIRS}
+)
+
+add_subdirectory(src/function)
+
+add_library(iceberg_extension
+        SHARED
+        ${ICEBERG_OBJECT_FILES}
+        src/iceberg_extension.cpp
+        ${PROJECT_SOURCE_DIR}/extension/delta/src/connector/delta_connector.cpp
```

Comment:
It is better to compile delta_connector as another cmake target instead of compiling all source files together.